frontend/src/components/tornjak-api-helpers.tsx

```diff
@@ -147,72 +109,32 @@ class TornjakApi extends Component<TornjakApiProp, TornjakApiState> {
   refreshLocalSelectorsState = (agentworkloadSelectorInfoFunc: {
     (globalAgentsWorkLoadAttestorInfo: AgentsWorkLoadAttestorInfo[]): void;
   }) => {
-    axios.get(GetApiServerUri("/api/tornjak/selectors/list"), { crossdomain: true })
+    axios.get(GetApiServerUri(apiEndpoints.tornjakSelectorsApi), { crossdomain: true })
       .then(response => {
         console.log(response.data)
         agentworkloadSelectorInfoFunc(response.data["agents"])
       })
       .catch((error) => showResponseToast(error, { caption: "Could not refresh local selector states." }))
   }
 
-  // populateTornjakAgentInfo returns tornjak info of requested agents including cluster name and selector
-  populateTornjakAgentInfo = (serverName: string,
-    agentworkloadSelectorInfoFunc: (globalAgentsWorkLoadAttestorInfo: AgentsWorkLoadAttestorInfo[]) => void,
-    inputData: string) => {
-    axios.post(GetApiServerUri("/manager-api/tornjak/agents/list/") + serverName, inputData,
-      {
-        crossdomain: true,
-      })
-      .then(response => {
-        agentworkloadSelectorInfoFunc(response.data["agents"]);
-      })
-      .catch((error) => showResponseToast(error, { caption: "Could not populate tornjak agent info." }))
-  }
-
   // populateLocalTornjakAgentInfo returns tornjak info of requested agents including cluster name and selector
   populateLocalTornjakAgentInfo = (
     agentworkloadSelectorInfoFunc: (globalAgentsWorkLoadAttestorInfo: AgentsWorkLoadAttestorInfo[]) => void,
     inputData: string
   ) => {
-    axios.post(GetApiServerUri("/api/tornjak/agents/list"), inputData, { crossdomain: true })
+    axios.post(GetApiServerUri(apiEndpoints.tornjakAgentsApi), inputData, { crossdomain: true })
```

Comment:
shouldn't this be a get?

Reply:
A different name is probably necessary there but that function updates agents including their cluster and selector info. There are some updates I need to make with that, good for this pr but will open a separate pr for it as their is a significant refactoring that can be done.

Reply:
I mean for the `axios.post` should it be an `axios.get`? the agents list api has become a get function in the v1 version

Reply:
Oh, you are right about that, pushed. Thanks.